orttraining/orttraining/training_ops/cuda/math/isfinite.cuh

```diff
@@ -8,12 +8,23 @@
 namespace onnxruntime {
 namespace cuda {
 
-template<typename T>
+template <typename T>
 __device__ __forceinline__ bool _IsFiniteScalar(const T value) {
```

Comment:
>_ [](start = 32, length = 1)

Generally, names starting with underscore are reserved names in C++. This was an unforntunate pattern started by someone.

Reply:
Sorry. I was the someone... lol

---
In reply to: [606355806](https://github.com/microsoft/onnxruntime/pull/7188#discussion_r606355806) [](ancestors = 606355806)